ui/src/shared/utils/checkQueryResult.ts

```diff
@@ -24,18 +24,7 @@ export const checkQueryResult = (file: string): void => {
     return
   }
 
-  const header = lines[0].split(',').map(s => s.trim())
-  const row = lines[1].split(',').map(s => s.trim())
-  const index = header.indexOf('error')
-
-  if (index === -1 || !row[index]) {
-    return
-  }
-
-  // Trim off extra quotes at start and end of message
-  const errorMessage = row[index].replace(/^"/, '').replace(/"$/, '')
-
-  throw new Error(errorMessage)
+  return
```

Comment:
the error is actually with the assumption made on line 18 (the error could be anywhere), and with the means of asserting an error type in line 29-36. the code should read more like:
```
if (!lines[0].test(/^,error,reference$/)) {
    return
}

const row = lines[1].split(',').map(s => s.trim())

throw new Error(`[${row[2]}] ${row[1]}`)
```